pyproject.toml

```diff
@@ -177,7 +177,10 @@ show_error_codes = true
 exclude = "airflow/"
 
 [[tool.mypy.overrides]]
-module = "airflow.*"
+module = [
```

Comment:
I think this and the mypy overrides below should be merged

Reply:
@schustmi Do you know why exactly we have two different `[[tool.mypy.overrides]]`?

Reply:
Was wondering the same thing, one of them explicitly has a  `follow_imports = "skip"` and the other one doesn't so with the release at 13 coming up I felt like it was save to leave this question for next time.

Reply:
https://mypy.readthedocs.io/en/stable/config_file.html - if you scroll down here it seems desired to have separate [[tool.mypy.overrides]]
instances that descrive different configurations for the associated modules

Reply:
The override section here is just a temporary fix for a specific version of airflow which should get removed once we update the dependency version. The second `follow_imports = "skip"` is intended for any dependency that is not airflow and should not be imported by mypy. However there are very few reasons to do this and we should use it carefully, usually putting it in our third section with `ignore_missing_imports=true` should be enough and is the preferred way if some type annotations are missing.

Reply:
added to the second as we need to skip the follow_imports for now as recommended by huggingface